youtube_dl/extractor/wdr.py

```diff
@@ -21,54 +21,41 @@ class WDRIE(InfoExtractor):
 
     _TESTS = [
         {
-            'url': 'http://www1.wdr.de/mediathek/video/sendungen/servicezeit/videoservicezeit560-videoplayer_size-L.html',
+            'url': 'http://www1.wdr.de/mediathek/video/sendungen/hier_und_heute/videostreetfoodpioniere100.html',
```

Comment:
Doesn't it make more sense to test the url from where `*-videoplayer_size-[LMS].html` gets extracted? That way the link extraction for `*-videoplayer_size-[LMS].html` and the media extraction are getting tested.